### PR TITLE
Documentation improvements

### LINKED ISSUE #4850: wrong PR links in changelog
for example, in https://doc.powerdns.com/md/changelog/#powerdns-authoritative-server-400-alpha3 the last 3 of these are wrong:
```
- [#3415](https://github.com/PowerDNS/pdns/pull/3415) pdnsutil: add clear-zone command
- [#3586](https://github.com/PowerDNS/pdns/pull/3415) Remove send-root-referral option
- [#3578](https://github.com/PowerDNS/pdns/pull/3415) Add disable-syslog option
- [#3733](https://github.com/PowerDNS/pdns/pull/3415) ALIAS improvements: DNSSEC and optional on-AXFR expansion of records
```

I suggest doing a nice grep to find them all.

### PR DESCRIPTION
**Pull request by pieterlexis**
### Short description
This PR fixes several documentation issues.

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled and tested this code
- [x] added regression tests

### CODE REVIEW

File: docs/markdown/end-of-life.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
 # End of life statements
-The currently supported release train of PowerDNS is 3.x. The development train is 4.x.  
-2.x is end of life.
+The currently supported release train of PowerDNS, for both the Authoritative Server and the Recursor is 4.x.
 
-## PowerDNS Authoritative Server 2.x
-21st of May 2015
+PowerDNS Authoritative Server 3.4 is considered legacy will only receive critical bug fixes and security fixes.
```
Comment (rgacogne) [inline docs/markdown/end-of-life.md]: how about "is considered legacy **and** will only receive.."?

File: docs/markdown/end-of-life.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+
+PowerDNS Authoritative Server 3.3 will only receive security fixes.
+
+PowerDNS Recursor 3.7 is considered legacywill only receive critical bug fixes and security fixes.
```
Comment (rgacogne) [inline docs/markdown/end-of-life.md]: this sentence is at least missing a space between "legacy" and "will", and perhaps a **and** as suggested above

File: docs/markdown/end-of-life.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
-and earlier are impacted by [PowerDNS Security Advisory 2012-01](https://doc.powerdns.com/md/security/powerdns-advisory-2012-01/), which means
-PowerDNS can be used in a Denial of Service attack.
+PowerDNS Authoritative Server 2.9.22 was released in January 2009.
+Because of its immense and durable popularity, some patch releases have been provided, the last one of which (2.9.22.6) was made available in January 2012.
 
-Although we have long been telling users that we can no longer support the
-use of 2.x, and urging upgrades to 3.x, with this statement we formally
-declare 2.x end of life.
+The 2.9.22.x series contains a number of probable and actual violations of the DNS standards.
+In addition, some behaviours of 2.9.22.x are standards conforming but cause interoperability problems in 2015.
```
Comment (Habbie) [inline docs/markdown/end-of-life.md]: s/in 2015/today/ ?

File: docs/markdown/end-of-life.md [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
+Finally, 2.9.22.4 and earlier are impacted by [PowerDNS Security Advisory 2012-01](https://doc.powerdns.com/md/security/powerdns-advisory-2012-01/), which means PowerDNS can be used in a Denial of Service attack.
 
-This means that any 2.x issues will not be addressed. This has been the case
-for a long time, but with this statement we make it formal.
+Although we have long been telling users that we can no longer support the use of 2.x, and urging upgrades to 3.x, with this statement we formally declare 2.x end of life.
```
Comment (Habbie) [inline docs/markdown/end-of-life.md]: upgrades to 4.x?

Comment (zeha) [inline docs/markdown/end-of-life.md]: or just `urging upgrading`?

**Pull request merged**